tests/Core/Config/SniffsExcludeArgsTest.php

```diff
@@ -62,47 +77,76 @@ public static function dataInvalidSniffs()
         ];
         $data      = [];
 
-        $messageTemplate = 'ERROR: The specified sniff code "%s" is invalid'.PHP_EOL.PHP_EOL;
+        $messageTemplate = 'ERROR: The specified sniff code "%s" is invalid'.PHP_EOL;
 
         foreach ($arguments as $argument) {
-            // An empty string is not a valid sniff.
-            $data[$argument.'; empty string'] = [
-                'argument' => $argument,
-                'value'    => '',
-                'message'  => sprintf($messageTemplate, ''),
-            ];
-
             // A standard is not a valid sniff.
             $data[$argument.'; standard'] = [
-                'argument' => $argument,
-                'value'    => 'Standard',
-                'message'  => sprintf($messageTemplate, 'Standard'),
+                'argument'   => $argument,
```

Comment:
As the data provider format has changed, the `@return` for the method needs to be updated.